ProteGO/Common/Bluetooth/BeaconId.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+/// This class represents valid Beacon ID value.
+class BeaconId {
```

Comment:
Dlaczego modele definiujesz jako klasy a nie struktury? Proponuję dla spójności korzystać ze struktur, jeśli to możliwe.

Pozostałe modele trzymamy w Common/DataModel.

Reply:
Zmienię to i przeniosę `BeaconId` oraz `ExpiringBeaconId` do `Common/DataModel`, ok?

Reply:
👍